packages/salesforcedx-vscode-core/test/vscode-integration/commands/auth/forceAuthLogout.test.ts

```diff
@@ -189,9 +189,15 @@ describe('Force Auth Logout Default', () => {
       'should not have reported an error'
     ).to.equal(false);
     expect(
-      notificationStub.called,
+      notificationStub.callCount,
       'should not have posted an error message'
-    ).to.equal(false);
+    ).to.equal(1);
```

Comment:
Thus looks like there's been a change i functionality.  Previously, the call to showInformationMessage wasn't getting called, but now it is?

Reply:
yeah, I think the functionality is the same but the context of the test changed.  B/c previously it was using `@salesforce/salesforcedx-utils-vscode/out/src/commands` to import and stub the notificationService it would only see calls to that service when referenced via that module path. B/c the `SfdxCommandletExecutor` in the sdfxCommendlet.ts file imported the notification service via `import { notificationService, ProgressNotification } from '../../notifications';` The stub would not be called as expected for the success flow.  Now that both of those NotificationService module loads come from the same module path we see all calls to that method.